frontend/src/pages/restartPage/connectivityWarningDialog/ConnectivityWarningDialog.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+
+import Dialog from "../../../components/atoms/dialog/Dialog";
+import Button from "../../../components/atoms/button/Button";
+
+import styles from "./ConnectivityWarningDialog.module.css";
+
+export type Props = {
+  active: boolean;
+  piTopIpAddress: string;
+  shouldMoveAwayFromAp: boolean;
+  onSkip: () => void;
+  onContinue: () => void;
+};
+
+export const getMessage = (shouldMoveAwayFromAp: boolean) => {
+  return (
+    <>
+      <span className={styles.dialogTitle}>{shouldMoveAwayFromAp ? "Reconnect to your Wi-Fi" : "Reconnect to the 'pi-top' network"} </span>
+    </>
+  );
+};
+
+export const getContent = (shouldMoveAwayFromAp: boolean) => {
+  if (shouldMoveAwayFromAp) {
+    return (
+      <>
+        Your pi-top is not connected to the same network as your device.
+        <br></br>
+        <br></br>
+        This probably means that your computer is connected to the 'pi-top' network and your pi-top was configured to connect to your Wi-Fi network in one of the previous steps.
+        <br></br>
+        <br></br>
+        Please, make sure you connect your computer to your home Wi-Fi network and refresh this page.
```

Comment:
```suggestion
        Please, make sure you connect your computer to your home Wi-Fi network and click refresh below.
```
if you're not on pi-top.local but an ip, it's important to click this button to be directed to the new ip